backend/src/main/java/com/google/blackswan/mock/SimpleAnomalyGenerator.java

```diff
@@ -34,22 +33,51 @@
 import java.nio.charset.StandardCharsets;
 import java.io.ByteArrayInputStream;
 import com.google.common.collect.ImmutableList;
+import com.google.blackswan.mock.filesystem.*;
```

Comment:
you're currently only using the LocalFileSystem right?

Reply:
yes, for all the code push to github, the `LocalFileSystem` is used in order to pass the unit tests as while github is running the tests it does not have access to Application Default Credential or `key.json`.